### PR TITLE
Revert "Removing Queue trigger F# template"

### PR DESCRIPTION
**Pull request by paulbatum**
This reverts commit 3f3a3860fbf60c0d418b2732ab106514a35d437a.

### CODE REVIEW

File: Templates/QueueTrigger-FSharp/metadata.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "defaultFunctionName": "QueueTriggerFSharp",
+    "description": "An F# function that will be run whenever a message is added to a specified Azure Storage queue",
+    "name": "QueueTrigger - F#",
+    "language": "F#",
+    "trigger": "QueueTrigger",
```
Comment (mathewc) [inline Templates/QueueTrigger-FSharp/metadata.json]: "trigger" property is obsolete

**Comment by mathewc** [comment]
Thanks :) :shipit:

File: Templates/QueueTrigger-FSharp/metadata.json (hunk 1 above)
Comment (soninaren) [inline Templates/QueueTrigger-FSharp/metadata.json]: The end 2 end script uses it. Lets keep it for now until i update the script.

Comment (paulbatum) [inline Templates/QueueTrigger-FSharp/metadata.json]: I believe the trigger metadata is being used by the CI automation, Naren can confirm. I double checked across the templates and there is some inconsistency between which ones have it and which ones dont. I will leave it here right now, we can do another pass to get this in order.

Comment (soninaren) [inline Templates/QueueTrigger-FSharp/metadata.json]: Sorry. E2E uses it from function.json. We can remove from here

Comment (paulbatum) [inline Templates/QueueTrigger-FSharp/metadata.json]: Ahh that makes sense.

**Comment by soninaren** [comment]
:+1: Looks good

**Pull request merged**